### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by khuantum**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ app.use(bodyParser.urlencoded({
   extended: true
 }));
 app.use(session({
-  secret: process.env.SECRET,
+  secret: "test",
```
Comment (ShawnCollinge) [inline app.js]: Should leave secret as the env file SECRET

File: views/partials/header.ejs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<head>
+  <meta charset="utf-8">
+  <title>Shawns website</title>
```
Comment (ShawnCollinge) [inline views/partials/header.ejs]: Change it to CS club website. That was my mistake on copy and paste from my website

**Review by ShawnCollinge — COMMENTED** [review]
Everything else looks great, just a couple fixes.

File: app.js (hunk 1 above)
Comment (Kneelawk) [reply app.js]: I don't know how this got here

File: views/partials/header.ejs (hunk 2 above)
Comment (Kneelawk) [reply views/partials/header.ejs]: Sure thing

File: app.js (hunk 1 above)
Comment (Kneelawk) [reply app.js]: https://github.com/Shoreline-CS-Club-2021-2022/website/commit/e1009d67458fc1a885353a35cd1eb83f676fd37c apparently. I am very confused.

Comment (Kneelawk) [reply app.js]: I'll fix it

**Comment by Kneelawk** [comment]
I should probably also add a clone command to the README for people who want to clone the backend instead of the frontend.

File: app.js (hunk 1 above)
Comment (Kneelawk) [reply app.js]: Ok, fixed.

File: views/partials/header.ejs (hunk 2 above)
Comment (Kneelawk) [reply views/partials/header.ejs]: Ok, fixed.

**Comment by Kneelawk** [comment]
Oh, I should also add extra IDE junk files to the .gitignore.

**Review by ShawnCollinge — APPROVED** [review]

**Pull request merged**